vscode-extension/README.md

```diff
@@ -9,9 +9,16 @@ The Numbat extension provides language support for the [Numbat](https://github.c
 - comment toggling
 - autosurrounding brackets
 
+## Installation
+
+To install this extension you will need to download the extension installation file (VSIX file), which can be found in the assets of each [numbat release](https://github.com/sharkdp/numbat/releases) after v1.14.0.  
```

Comment:
> which can be found in the assets

can it? Am I missing something?

Reply:
I thought that was the approach we agreed on for the distribution of the add-in. I might have misunderstood but that at least was my understanding when writing this paragraph.

Reply:
Ah, sorry — yes. But we haven't implemented that (in `.github/workflows/ci.yml`) yet :smile: To be honest, I don't even know what VSIX is and/or how to generate it.

Reply:
Aah fair. VSIX is just a visual studio code installation executable. So just a packaging format for code extensions to share them. It can be generated via npm with this script: https://github.com/sharkdp/numbat/blob/master/vscode-extension/package.json#L38

I didn't think about changing the GitHub Action but I could probably add it as a step without a problem.